pyogrio/geopandas.py

```diff
@@ -347,6 +349,8 @@ def write_dataframe(
 
         if geometry_type is None:
             geometry_type = tmp_geometry_type
+            if has_z:
+                geometry_type = f"2.5D {geometry_type}"
```

Comment:
https://github.com/geopandas/pyogrio/commit/8679f3e2a07b62940b8bd9f55d1b4c690acc76ef added the versions with " Z" suffix as well? Do we want to use those ourselves as well? (I personally find that easier to read than this strange "2.5D", but of course that's what GDAL itself uses it)

Reply:
I used this notation because this is also the way `pyogrio.read_info` will report it... so at first sight it seems less confusing to me to stick to it... but...

Reply:
We could also change what is presented on read to make them consistent, and stick to GEOS-style "Z" suffixes throughout.  Perhaps to keep this moving ahead, stick with `2.5D` for consistency, then we revisit this comprehensively later?

Reply:
Making the "Z" syntax consistent is in my opinion also a good option. It's a breaking change, but few people will notice. 

In fiona they apparently use a third variant for the same thing: e.g. "3D POINT": https://fiona.readthedocs.io/en/latest/manual.html#geometry-types

But, it's only internal here, so indeed not "urgent".